website/docs/r/cluster.html.markdown

```diff
@@ -348,6 +348,13 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri` - Base connection string for the cluster. Atlas only displays this field after the cluster is operational, not while it builds the cluster.
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
+* `connection_strings` - Set of connection strings that your applications use to connect to this cluster.
+    - `connection_strings.standard` -  `mongodb://` A required prefix to identify that this is a string in the standard connection format.
```

Comment:
- `connection_strings.standard` -  Public mongodb:// connection string for this cluster.

Reply:
changes has been made, thank you melissa.